Resources/translations/content_type.en.xlf

```diff
@@ -22,10 +22,42 @@
         <source>content_type.group.edit</source>
         <target>Edit</target>
       </trans-unit>
+      <trans-unit id="2cf33e780bfbad13ad16fcf5634ec70c">
+        <source>content_type.group.new</source>
+        <target>Create a ContentType group</target>
+      </trans-unit>
       <trans-unit id="7b24488c5d8ce01a3dd1a49ca7381d0b" resname="content_type.group">
         <source>content_type.group</source>
         <target>Content type group</target>
```

Comment:
Perhaps offtopic, but since you use "ContentType group" for all the new entries it would be nice to make this one consistent.